syntax = "proto3";
import "common.proto";

service Showtime {
    rpc GetShowtimeByDate(ShowtimeDate) returns (ShowtimeData) {}
    rpc GetShowtimes(Empty) returns (stream ShowtimeData) {}
}

message ShowtimeDate {
    string date = 1;
}

